### PR TITLE
Adapt model for health check commands

### PR DESCRIPTION
**Pull request by jmalvarezf**
Hi,

I've modified the model for the Marathon API in DC/OS 1.8 so that it reflects the changes they have done to the health check information. It impacted our deployment when we changed the health check to command in DC/OS.

Hope it helps,

Jose Maria.

### CODE REVIEW

File: common/mesos-membership-scheme/src/main/java/org/wso2/carbon/clustering/mesos/client/model/marathon/v2/Command.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.wso2.carbon.clustering.mesos.client.model.marathon.v2;
```
Comment (imesh) [inline common/mesos-membership-scheme/src/main/java/org/wso2/carbon/clustering/mesos/client/model/marathon/v2/Command.java]: Would you mind adding the license header?

File: common/mesos-membership-scheme/src/main/java/org/wso2/carbon/clustering/mesos/client/model/marathon/v2/Command.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    private String value;
+
+    public String getValue() {
+        return value;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
```
Comment (imesh) [inline common/mesos-membership-scheme/src/main/java/org/wso2/carbon/clustering/mesos/client/model/marathon/v2/Command.java]: May be we can remove the additional blank line here. Thanks!

**Review by imesh — CHANGES_REQUESTED** [review]

**Comment by jmalvarezf** [comment]
Hello @imesh,

It's done!

Thank you,

Jose Maria.

**Review by imesh — APPROVED** [review]

**Comment by imesh** [comment]
Thank you very much Jose for your contribution! This is now merged to the master branch.

**Pull request merged**